honeybee_energy/properties/extension.py

```diff
@@ -197,4 +197,36 @@ class EnergyMaterialNoMassProperties(_EnergyProperties):
 
 
 class EnergyMaterialVegetationProperties(_EnergyProperties):
-    """EnergyMaterialVegetation properties to be extended by extensions."""
\ No newline at end of file
+    """EnergyMaterialVegetation properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialGlazingsProperties(_EnergyProperties):
+    """EnergyWindowMaterialGlazing properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialSimpleGlazSysProperties(_EnergyProperties):
+    """EnergyWindowMaterialSimpleGlazSys properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialGasProperties(_EnergyProperties):
+    """EnergyWindowMaterialGas properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialGasMixtureProperties(_EnergyProperties):
+    """EnergyWindowMaterialGasMixture properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialGasCustomMixtureProperties(_EnergyProperties):
+    """EnergyWindowMaterialGasCustom properties to be extended by extensions."""
```

Comment:
Maybe this is related to my other comment but the name of the class is `EnergyWindowMaterialGasCustom`. Is there a reason why you added the word `Mixture` in here?

Reply:
whoops - yes that is just a typo from too much copy/pasting. I have pushed an update to remove this typo.